### PR TITLE
add ultraliser package

### PR DESCRIPTION
**Pull request by ppodhajski**

### CODE REVIEW

**Review by matz-e — APPROVED** [review]
Shouldn't it be "Ultralizer"?

**Comment by ppodhajski** [comment]
> Shouldn't it be "Ultralizer"?
Good question for @marwan-abdellah  ;-)

should it be in gnu-stable-serial? it does not require python and I guess there is some additional check now as it fails:

`12:13:59  ==> Error: ultraliser does not depend on python`

File: deploy/packages/bbp-packages.yaml [hunk 1]
```diff
@@ -35,6 +35,7 @@ packages:
       - py-entity-management@1.0.12
       - py-jupyter-notebook@4.2.3
       - brion@3.0.0 +python
+      - ultraliser
```
Comment (matz-e) [inline deploy/packages/bbp-packages.yaml]: Your intuition is right. This should actually be in `gnu-stable` if it does not depend on Python. See the very bottom of this file.

**Review by matz-e — CHANGES_REQUESTED** [review]

**Review by matz-e — APPROVED** [review]

**Comment by matz-e** [comment]
Let's nix nix!

**Pull request merged**